v1/firebase/adapter.js

```diff
@@ -40,7 +40,18 @@ function apiGetTrack(trackId) {
 
 function apiGetChannel(channelId) {
 	return apiGet(`channels/${channelId}`).then(snapshot => {
-		return serializeChannel(snapshot.val(), channelId);
+		var channel = serializeChannel(snapshot.val(), channelId);
+
+		// Replace `images` with a single `thumbnail` URL.
+		var images = Object.keys(channel.images);
+		var lastImage = images[images.length - 1];
```

Comment:
No :( Long story.. We have `channel.hasMany('images')` but actually we always use the latest image as the thumbnail. Have not managed to delete old ones from Cloudinary when you upload a new photo on Radio4000.